cmd/config-sync-controllers/main.go

```diff
@@ -91,7 +91,7 @@ func main() {
 	options.BindLeaderElectionFlags(&leaderElectionConfig, pflag.CommandLine)
 	pflag.Parse()
 
-	ctrl.SetLogger(klogr.New().WithName("CCMOCloudConfigSyncController"))
+	ctrl.SetLogger(klogr.New().WithName("CCMOConfigSyncController"))
```

Comment:
I know this was broken before, but, it should be
```suggestion
	ctrl.SetLogger(klogr.New().WithName("CCCMOConfigSyncController"))
```